clientSide/src/parse-functions/carTableFunctions.js

```diff
@@ -3,14 +3,16 @@ import {
   ColumnLabels,
   ErrorLabels,
 } from "../text-labels/parse-labels";
+import Parse from "parse";
 
-export const setCarElements = async ({ setCars, setError }) => {
+export const setCarElements = async (setCars, setError) => {
   const Car = Parse.Object.extend(ClassnameLabels.car);
   const query = new Parse.Query(Car);
   query.include(ColumnLabels.car.group);
   query.include(ColumnLabels.car.rentalOffice);
   query.include(ColumnLabels.car.state);
   query.include(ColumnLabels.car.id);
+  query.notEqualTo(ColumnLabels.car.licenseNo, "no car assigned");
```

Comment:
extract the "no car assigned" to its own label in the parse-labels.js file and name it something like: HardcodedValueLabels